homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -54,6 +55,7 @@
     vol.Optional(CONF_INCL_FILTER): cv.string,
     vol.Optional(CONF_EXCL_FILTER): cv.string,
     vol.Optional(CONF_REALTIME, default=DEFAULT_REALTIME): cv.boolean,
+    vol.Optional(CONF_UNITS): vol.In(UNITS)
```

Comment:
almost forgot. I still think we should get units from ha as default but allow user to override.

Reply:
Yes, that's what this does

Reply:
ah sorry. awesome. Hopefully this get merged before release.

Reply:
I'm not sure, I thought you were reviewing this.  Haha.  It may get put it or not?  Don't know.  Either way, you can use it as a custom component.  I've tested it pretty thoroughly.

Reply:
Nah I just help out reviewing/coding/etc. I don't have merge rights.